p2pserver/common/p2p_common.go

```diff
@@ -111,50 +93,31 @@ type PeerAddr struct {
 	Port     uint16   //sync port
 	//todo remove this legecy field
 	ConsensusPort uint16 //consensus port
-	ID            uint64 //Unique ID
+	ID            PeerId //Unique ID
 }
 
 //const channel msg id and type
 const (
-	VERSION_TYPE     = "version"    //peer`s information
-	VERACK_TYPE      = "verack"     //ack msg after version recv
-	GetADDR_TYPE     = "getaddr"    //req nbr address from peer
-	ADDR_TYPE        = "addr"       //nbr address
-	PING_TYPE        = "ping"       //ping  sync height
-	PONG_TYPE        = "pong"       //pong  recv nbr height
-	GET_HEADERS_TYPE = "getheaders" //req blk hdr
-	HEADERS_TYPE     = "headers"    //blk hdr
-	INV_TYPE         = "inv"        //inv payload
-	GET_DATA_TYPE    = "getdata"    //req data from peer
-	BLOCK_TYPE       = "block"      //blk payload
-	TX_TYPE          = "tx"         //transaction
-	CONSENSUS_TYPE   = "consensus"  //consensus payload
-	GET_BLOCKS_TYPE  = "getblocks"  //req blks from peer
-	NOT_FOUND_TYPE   = "notfound"   //peer can`t find blk according to the hash
-	DISCONNECT_TYPE  = "disconnect" //peer disconnect info raise by link
+	VERSION_TYPE       = "version"     //peer`s information
+	VERACK_TYPE        = "verack"      //ack msg after version recv
+	GetADDR_TYPE       = "getaddr"     //req nbr address from peer
+	ADDR_TYPE          = "addr"        //nbr address
+	PING_TYPE          = "ping"        //ping  sync height
+	PONG_TYPE          = "pong"        //pong  recv nbr height
+	GET_HEADERS_TYPE   = "getheaders"  //req blk hdr
+	HEADERS_TYPE       = "headers"     //blk hdr
+	INV_TYPE           = "inv"         //inv payload
+	GET_DATA_TYPE      = "getdata"     //req data from peer
+	BLOCK_TYPE         = "block"       //blk payload
+	TX_TYPE            = "tx"          //transaction
+	CONSENSUS_TYPE     = "consensus"   //consensus payload
+	GET_BLOCKS_TYPE    = "getblocks"   //req blks from peer
+	NOT_FOUND_TYPE     = "notfound"    //peer can`t find blk according to the hash
+	FINDNODE_TYPE      = "findnode"    // find node using dht
```

Comment:
http://www.bittorrent.org/beps/bep_0005.html ; use  "ping" / "get_peers" / "find_node" / "announce_peer" ?

Reply:
I think the name is consistent with `getblocks`, `getheaders`. the implementation details are diferent with bittorrent, such as different message fields.